libraries/zookeeper_helper.rb

```diff
@@ -52,29 +48,29 @@ def setup_helper
         id = id + 1
       end
 
-      raise "Unable to find server [#{node["fqdn"]} in servers attribute #{node["zookeeper"]["servers"]}" if @zookeeper_myid.nil?
+      raise "Unable to find server [#{node["fqdn"]} in servers attribute #{node['apache_zookeeper']["servers"]}" if @zookeeper_myid.nil?
     end
 
   end
 
-  def zookeeper_base *files
-    File.join node["zookeeper"]["base_directory"], *files
-  end
-
-  def zookeeper_bin *files
-    zookeeper_base "bin", *files
-  end
-
-  def zookeeper_conf *files
-    zookeeper_base "conf", *files
-  end
-
-  def zookeeper_tar_path
-    File.join Chef::Config[:file_cache_path], "zookeeper-#{node["zookeeper"]["version"]}.tar.gz"
-  end
-
+#  def zookeeper_base *files
+#    File.join node['apache_zookeeper']["base_directory"], *files
+#  end
+#
+#  def zookeeper_bin *files
+#    zookeeper_base "bin", *files
+#  end
+#
+#  def zookeeper_conf *files
+#    zookeeper_base "conf", *files
+#  end
+#
+#  def zookeeper_tar_path
+#    File.join Chef::Config[:file_cache_path], "zookeeper-#{node['apache_zookeeper']["version"]}.tar.gz"
+#  end
+#
```

Comment:
I need to review this a bit more. I think I just use the helper that creates `myid`. Is that suppose to generate anything in a standalone config?